crs/v1alpha2/cluster.yaml

```diff
@@ -20,5 +20,5 @@ kind: BareMetalCluster
 metadata:
   name: ${CLUSTER_NAME}
 spec:
- apiEndpoint: http://192.168.111.249:6443
+ apiEndpoint: http://${CLUSTER_APIENDPOINT_IP}:6443
```

Comment:
Hm, this URL doesn't look like it's constructed in a way that's compatible with IPv6. You can see an example of how we do this with `$CLUSTER_URL_HOST`, basically IPv6 addresses in the host part of a URL need the surrounded by square brackets.

Reply:
Thanks for pointing this out. I have changed the code accordingly

Reply:
Thanks for point this out. I have changed the code accordingly